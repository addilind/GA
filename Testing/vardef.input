a := 5;
a;
2*a;
b := 0.1;
b;
10.0*b;