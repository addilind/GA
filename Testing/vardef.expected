5\n10\n0.1\n1